fixtures/manufacturers.json

```diff
@@ -406,6 +406,10 @@
     "website": "https://prolights.it/",
     "rdmId": 5584
   },
+  "purelight": {
+    "name": "PURElight",
```

Comment:
The logos all say "PURElight by lightmaXX", and OFL already has a "lightmaxx" manufacturer.

Is this considered the same manufacturer or different?  I don't know.  Every company seems to be buying up every other company these days.